TYPO3.Neos/Configuration/Settings.yaml

```diff
@@ -302,24 +302,24 @@ TYPO3:
       defaultEditPreviewMode: 'inPlace'
       editPreviewModes:
         # Live mode is only configured here for consistency. You shouldn't change it.
-        live:
+        'TYPO3.Neos:live':
```

Comment:
> # Live mode is only configured here for consistency. You shouldn't change it.

But if we can not change it, why not removing this mode from the settings and apply this mode only in code ? And make the "live" key a reserved key.